src/app/component/dashboard/dashboard.component.html

```diff
@@ -1,4 +1,22 @@
 <div class="dashboard">
+<<<<<<< HEAD
```

Comment:
Merge conflict remnants need to be cleaned up.

Reply:
Sorry, saw this one after I did the PR.. resolved conflict.